docs/ddl-of-carbondata.md

```diff
@@ -646,6 +647,28 @@ CarbonData DDL statements are documented here,which includes:
 
 ## TABLE MANAGEMENT  
 
+### DESCRIBE COMMAND
+
+Describe column of table and visualize its structure with child fields.
+  ```
+  DESCRIBE COLUMN fieldname[.nestedFieldNames] ON [db_name.]table_name;
+
+  Example: DESCRIBE COLUMN channelsId ON carbonTable;
```

Comment:
Can add some examples to query Array child, since have to provide arr_name.item

Reply:
Done